packages/core/src/components/portal/_portal.scss

```diff
@@ -34,7 +34,3 @@ The children of a `Portal` component are appended to the `<body>` element.
 
 Styleguide components.portal.js
 */
-
-.pt-portal {
-  display: inline-block;
-}
```

Comment:
They being the portal? Or the reporter of the issue that this fixes? Are we sure the portal will always use a div?